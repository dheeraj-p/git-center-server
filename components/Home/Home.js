import Repositories from '../Repositories/Repositories';
import Header from '../Header/Header';
import CreateRepository from '../CreateRepository/CreateRepository';
import AddSSHKey from '../AddSSHKey/AddSSHKey';

export default function Home() {
  return (
    <div>
      <Header />
      <div className="container-fluid">
        <div className="row">
          <div className="col-md">
            <div className="d-flex flex-column">
              <CreateRepository />
              <AddSSHKey />
            </div>
          </div>
          <div className="col-md">
            <Repositories />
          </div>
        </div>
      </div>
    </div>
  );
}
