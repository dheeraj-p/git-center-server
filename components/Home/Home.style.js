import css from 'styled-jsx/css';
import colors from '../../styles/colors';

const HomeStyles = css`
  .add-fields {
    display: flex;
    flex-direction: column;
    margin: 20px;
  }

  .home-container {
    display: flex;
    width: 100vw;
  }
`;

export default HomeStyles;
