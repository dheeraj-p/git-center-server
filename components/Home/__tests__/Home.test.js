import { shallow } from 'enzyme';
import Home from '../Home';

describe('Home', () => {
  it('should render properly', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
