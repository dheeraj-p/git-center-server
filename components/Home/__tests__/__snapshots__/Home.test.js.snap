// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render properly 1`] = `
"<div>
  <Header />
  <div className=\\"container-fluid\\">
    <div className=\\"row\\">
      <div className=\\"col-md\\">
        <div className=\\"d-flex flex-column\\">
          <CreateRepository />
          <AddSSHKey />
        </div>
      </div>
      <div className=\\"col-md\\">
        <Repositories />
      </div>
    </div>
  </div>
</div>"
`;
